packages/cli/cloud/src/create-project/utils/set-default-name.ts

```diff
@@ -0,0 +1,14 @@
+import { DistinctQuestion } from 'inquirer';
+import { ProjectAnswers } from '../../types';
+
+export function setDefaultName(
+  newDefaultName: string,
+  questions: ReadonlyArray<DistinctQuestion<ProjectAnswers>>,
+  defaultValues: Partial<ProjectAnswers>
+) {
+  defaultValues.name = newDefaultName;
+  const nameQuestion = questions.find((q) => q.name === 'name');
+  if (nameQuestion) {
+    nameQuestion.default = newDefaultName;
+  }
```

Comment:
Can you avoid mutating parameters, please?
I believe we want to go towards functional programming and pure functions. 👍

Reply:
Nice thought, function updated!